technical/indicators/overlap_studies.py

```diff
@@ -18,19 +19,36 @@ def bollinger_bands(
     stdv: int = 2,
     field: str = "close",
     colum_prefix: str = "bb",
+    ma_type: str = "sma",
 ) -> DataFrame:
     """
-    Bollinger bands, using SMA.
-    Modifies original dataframe and returns dataframe with the following 3 columns
+    Bollinger bands, using Moving Average.
+    Copying original dataframe and returns dataframe with the following 3 columns
         <column_prefix>_lower, <column_prefix>_middle, and <column_prefix>_upper,
     """
-    rolling_mean = dataframe[field].rolling(window=period).mean()
-    rolling_std = dataframe[field].rolling(window=period).std(ddof=0)
```

Comment:
hm, ddof=0 intentionally "dropped" ?

Reply:
I'm matching it to qptylib BB, where they use the default ddof of 1. Do we want to use 0 or 1?

Reply:
i noticed it as difference in the command diff - i didn't yet look what difference it actually makes :laughing: - hoping it was intentional and you'd tell me "well it's because ..." :rofl:

Reply:
A simple test done shows a difference of 0.022%
using ddof 1 is 59354.72414351207
ddof 0 is 59367.93181841272

Reply:
Google give me this. According to this, ddof of 1 is better

https://www.allendowney.com/blog/2024/06/08/which-standard-deviation/

Reply:
sounds like a plan 😆